src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -207,8 +207,8 @@ class RemoteServicesMockServer(port:Int) {
 
     mockServer.when(
       request()
-        .withMethod("DELETE")
-        .withPath("/workflow/already_terminal_workflow")
+        .withMethod("POST")
+        .withPath("/workflows/v1/already_terminal_workflow/abort")
```

Comment:
Actually Cromwell would return 400s for all of these tests as these workflow IDs wouldn't parse as UUIDs